modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,45 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-developer-perspective.adoc
+
+[id="interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+The *Pipelines* view in the *Developer* perspective lists all the pipelines in a project along with details, such as the namespace in which the pipeline was created, the last pipeline run, the status of the tasks in the pipeline run, the status of the pipeline run, and the time taken for the run.
+
+[Discrete]
+=== Procedure
+. In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down to see the pipelines in that project.
++
+image::op-pipeline-list.png[Pipelines in the Developer perspective]
++
+. Click on the required pipeline to see the *Pipeline Details* page. It provides a visual representation of all the serial and parallel tasks in the pipeline. The tasks are also listed to the bottom-right of the page. You can click the listed *Tasks* to view the details of the task.
++
+image::op-pipeline-details.png[Pipeline details]
++
+. Optionally, in the *Pipeline Details* page :
+
+* Click *Pipeline Runs* tab to see the completed, running, or failed runs for the pipeline. You can use the Options menu (icon image)  to: stop a running pipeline, rerun a pipeline using the same parameters and resources as that of the previous pipeline execution, or delete a pipeline run.
+* Click *Parameters* tab to see the parameters defined in the Pipeline. You can also add or edit additional parameters as required.
+* Click *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
+
+. Click *Actions* and select *Start* to start the pipeline. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the pipeline definition.
```

Comment:
The `Start Pipeline` dialog is displayed *only* if some input is needed. Maybe you could write it differently so that it's clear that in some situations it won't be displayed.

Reply:
When I tested this, I saw the dialog box. I will test it again.

Reply:
@ppitonak on 4.4 cluster, I still see the Start modal. Can you please let us know in which situations it does not show?

Reply:
@Preeticp - Can you pls confirm if this is valid?